### DISCUSSION TITLE
Any plan to support OSPF?

### DISCUSSION THREAD

**Discussion by mirkomaffioli** [thatmattlove/hyperglass#147] [Ideas]
First of all, great product guys!
Do you have any plan to support OSPF query (on Cisco IOS in particular) ?

Thanks,
Mirko

**Comment by netravnen** [comment]
Why not use the custom commands for this? (~Overriding Built-In Commands)

- https://hyperglass.dev/docs/commands#overriding-built-in-commands

Override this,

- bgp_route

And disable the commands you will not need,

- bgp_aspath
- bgp_community

**Reply by netravnen** [reply]
With command override, you can define whichever "show route (...)" command you would like to see the result of.
